shared/media-accelerator/get-started/_get-started-sdk.mdx

```diff
@@ -0,0 +1,168 @@
+
+## Understand the tech
+
+The light-weight <Vpd k="SDK" /> is ideal for IoT applications due to its low power consumption. The SDK provides the following performance benefits:
+
+* **Small SDK size**: The increase in app size is less than 400 KB after integration.
+* **Low memory usage**: When the SDK is simultaneously sending and receiving 320x240 H.264 video data, memory usage is less than 2 MB.
+* **High connectivity**: Under normal conditions, connectivity is greater than 99%.
+* **Low bandwidth usage**: A maximum of 50 Mbps for a user in an RTC channel.
+* **Network adaptability**:	Non-aware recovery from up to 50% packet loss.
+
+The following figure shows the workflow you need to integrate <Vpd k="SDK" /> into your <Vpl k="CLIENT" />.
+
+![Media accelerator](/images/media-accelerator/media-accelerator-get-started.svg)
+
+## Prerequisites
+
+In order to follow this procedure you must have:
+
+<PlatformWrapper platform="linux-cpp">
+* A device running Linux Ubuntu 14.04 or above; 18.04+ is recommended.
+* At least 2 GB of memory.
+* `cmake` 3.6.0 or above.
+</PlatformWrapper>
+
+<Prerequites />
+
+## Project setup
+
+<ProjectSetup />
+
+
+<ProjectImplement />
+
+## Test your implementation
+
+To test the <Vpd k="PRODUCT" /> implementation:
+
+1. [Generate a temporary token](../reference/manage-agora-account#generate-a-temporary-token) in <Vg k="CONSOLE" />.
+
+2. In your browser, navigate to the <Link target="_blank" to="{{Global.DEMO_BASIC_VIDEO_CALL_URL}}"><Vg k="COMPANY" /> web demo</Link> and update _App ID_, _Channel_, and _Token_ with the values for your temporary token, then click **Join**.
+
+<ProjectTest />
+
+## Reference
+
+This section contains information that completes the information in this page, or points you to documentation that explains other aspects to this product.
+
+### Intercommunication with RTC SDKs
+
+If your business scenarios require not only high-quality audio and video transmission, but also easy integration, low memory usage, and low power consumption, such as smart cameras and smart doorbells, Agora recommends that you use the RTC SDK (v4.x) with the <Vpd k="SDK" />.
+
+The following table shows the data formats supported when the <Vpd k="SDK" /> communicates with the RTC SDK (v4.x):
+
+| SDK Type | <Vpd k="SDK"/> |
+|----|---------------|
+| RTC Native/third-party framework SDK (v4.x) | <ul><li>Audio: G722, G711, Opus, AAC</li><li>Video: H.264, JPEG</li></ul> |
+| RTC Web SDK (v4.x) | <ul><li>Audio: G722, G711, Opus</li><li>Video: H.264</li></ul> |
+| RTC applet SDK | <ul><li>Audio: G711, Opus</li><li>Video: H.264, JPEG</li></ul> |
+
+The sending and receiving ends can realize intercommunication only if the set data format is consistent and complies with the above table.
+
+The following sections describe how to set the audio and video data format in the SDK.
+
+#### Setup <Vpd k="SDK" />
+
+**Audio**
```

Comment:
Not sure why this is here. This info should be in the code for the example, or in the call quality guide. Can you have a bit of a think.

Reply:
Moving some of it to the call quality doc. 
I will add the rest as a new doc titled "Intercommunication with RTC SDKs" under Reference docs.